mne_bids/utils.py

```diff
@@ -726,6 +726,7 @@ def _read_events(events_data, event_id, raw, ext):
         The data as MNE-Python Raw object.
     ext : str
         The extension of the original data file.
+    %(verbose)s
```

Comment:
this will not work as you don't have a fill_doc decorator on the function _read_events

Reply:
Added `@verbose` decorator. Not sure if that is the right way: just copied the decorator from `read_events`, `find_events`, and `events_from_annotations`.

Reply:
Just document it manually. You might need the function `fill_docs` to make it work.

Reply:
Documented manually. Copied the text from `mne.utils.docs.docdict['verbose']`.